src/css/style.scss

```diff
@@ -3,39 +3,45 @@
   z-index: 0;
 }
 
+//nav
 nav {
+  align-items: center;
   display: flex;
   flex-direction: column;
-  align-items: center;
-}
-
-button {
-  width: 12em;
 }
 
-body {
-  font-family: 'Open Sans';
-  margin: 0;
-  background-color: #4d4e4f;
-  height: auto;
-}
-
-.pop-up-card {
-  width: 50em;
-  height: 20em;
-  background-color: green;
-  margin-left: 3em;
-  z-index: 1;
-  position: absolute;
-  top: 145px;
-  left: 200px;
+.nav-button-container {
+  padding-bottom: 1em;
 }
 
 h1 {
-  text-align: center;
   color: #ffffff;
   height: 80px;
   padding-top: 10px;
+  text-align: center;
+}
+
+button {
+  background-color: steelblue;
+  border-radius: 1em;
+  color: white;
+  cursor: pointer;
+  font-weight: bold;
+  height: 3em;
+  width: 14em;
+}
+
+button:hover {
```

Comment:
nice hover effects! looks good.

Reply:
Love the button changes. Great job Michelle.